Public/Src/Tools/Execution.Analyzer/Analyzers.Core/Xldb/XLGToDBAnalyzer.cs

```diff
@@ -163,15 +165,17 @@ internal sealed class XLGToDBAnalyzerInner : Analyzer
         private bool m_accessorSucceeded;
         private KeyValueStoreAccessor m_accessor;
         private Stopwatch m_stopWatch;
-        private int m_eventCount;
         private int m_eventSequenceNumber;
-        private Dictionary<Scheduler.Tracing.ExecutionEventId, EventCountByTypeValue> m_eventCountByType = new Dictionary<Scheduler.Tracing.ExecutionEventId, EventCountByTypeValue>();
+        private int m_eventCount;
+        private ConcurrentDictionary<DBStoredTypes, DBStorageStatsValue> m_dBStorageStats = new ConcurrentDictionary<DBStoredTypes, DBStorageStatsValue>();
         private string[] m_additionalColumns = { XldbDataStore.EventColumnFamilyName, XldbDataStore.PipColumnFamilyName, XldbDataStore.StaticGraphColumnFamilyName };
 
         private ConcurrentBigMap<Utilities.FileArtifact, HashSet<uint>> m_fileConsumerMap = new ConcurrentBigMap<Utilities.FileArtifact, HashSet<uint>>();
         private ConcurrentBigMap<Utilities.FileArtifact, uint> m_dynamicFileProducerMap = new ConcurrentBigMap<Utilities.FileArtifact, uint>();
         private ConcurrentBigMap<Utilities.DirectoryArtifact, HashSet<uint>> m_directoryConsumerMap = new ConcurrentBigMap<Utilities.DirectoryArtifact, HashSet<uint>>();
 
+        private NameExpander m_nameExpander = new NameExpander(108631);

```

Comment:
>108631 [](start = 63, length = 6)

add comment explaining how this value was selected

Reply:
also, maybe extract is a const?

---
In reply to: [321887446](https://github.com/microsoft/BuildXL/pull/836#discussion_r321887446) [](ancestors = 321887446)